ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -741,20 +768,33 @@ public class BottomSheetController: UIViewController {
 
     // Height of the sheet in the fully expanded state
     private var expandedSheetHeight: CGFloat {
+        guard isExpandable else {
+            return collapsedSheetHeight
+        }
+
         let height: CGFloat
-        let minHeight = headerContentHeight + (isExpandable ? ResizingHandleView.height : 0.0)
-        let maxHeight: CGFloat = view.frame.height - view.safeAreaInsets.top - Constants.minimumTopExpandedPadding
+        let maxHeight = view.frame.height - view.safeAreaInsets.top - Constants.minimumTopExpandedPadding
 
         if preferredExpandedContentHeight == 0 {
             height = maxHeight
         } else {
-            let idealHeight = minHeight + preferredExpandedContentHeight + view.safeAreaInsets.bottom
+            let idealHeight = currentResizingHandleHeight + headerContentHeight + preferredExpandedContentHeight + view.safeAreaInsets.bottom
             height = min(maxHeight, idealHeight)
         }
 
-        // Min height is required for cases when view.frame is .zero (before the initial layout pass)
+        // One case when the lower bound is required is when view.frame is .zero (like before the initial layout pass)
         // This gives the sheet some space to layout in so the layout engine doesn't complain.
-        return max(minHeight, height)
+        return max(collapsedSheetHeight, height)
+    }
+
+    // Height of the sheet in collapsed state
+    private var collapsedSheetHeight: CGFloat {
+        let visibleContentHeight = (collapsedContentHeight > 0) ? collapsedContentHeight : headerContentHeight
+        return currentResizingHandleHeight + visibleContentHeight + view.safeAreaInsets.bottom
+    }
+
+    private var currentResizingHandleHeight: CGFloat {
```

Comment:
Minor: Do we have some convention on preferring computed vars vs. funcs? Or what the tradeoffs are?

Reply:
Not that I'm aware of. I'd be very surprised if there were actual perf differences. Do you prefer a func here?

Reply:
I don't know of any differences or have an preferences - but it feels like we should since otherwise it's 2 identical ways of doing the same thing? Can talk about outside the scope of this PR though.

Reply:
I think the undocumented pattern we're following is that we use computed properties when:
1. no parameters are needed
2. there is a return type
3. the implementation does not imply actions, but calculates a value to be returned

**Examples of computed properties:**
 - var leadingViewPadding: Int
 - var window: UIWindow?
 
 **Examples of funcs with no parameters:**
 - func updateForCurrentTheme()
 - func reload()
 - func setupDraggingWindow()